frontend/src/metabase-lib/v1/expressions/complete/util.ts

```diff
@@ -49,27 +49,21 @@ export function expressionClauseCompletion(
 
   return {
     type,
-    label: suggestionText(clause),
+    label: clause.displayName,
     displayLabel: clause.displayName,
     icon: "function",
     matches,
   };
 }
 
-const suggestionText = (func: MBQLClauseFunctionConfig) => {
-  const { displayName, args } = func;
-  const suffix = args.length > 0 ? "(" : " ";
-  return displayName + suffix;
-};
-
 function getSnippet(helpText: HelpText) {
   const args = helpText.args
     ?.filter((arg) => arg.name !== "…")
     ?.map((arg) => "${" + (arg.template ?? arg.name) + "}")
     .join(", ");
 
   if (!args || args.length < 1) {
-    return `${helpText.structure}`;
+    return `${helpText.structure}()`;
```

Comment:
should the `structure` include `()`?

Reply:
No AFAICT, `structure` contains just the name as it is printed in the expression.

It is the same as `displayName` in `config.ts`. 
I would like to merge `helper-text-strings.ts` and `config.ts` in the future to remove this duplication.